README.md

```diff
@@ -76,6 +77,10 @@ or
 1. if run with `-j/--json` it will log output json formatted so that its output
    can be used by other programs easily
 
+`kubeaudit` supports using manual audit configuration provided by the user, use the command
+`kubeaudit -f/--manifest /path/to/manifest.yml -k/--kubeConfig /path/to/config.yml`
```

Comment:
I wouldn't name it kubeconfig because that's a thing already in kubernetes and in kubeaudit. how about `(kube)auditconfig`?